packages-nextgen/kestrel_core/src/kestrel/frontend/compile.py

```diff
@@ -61,17 +62,61 @@ def _create_comp(field: str, op: str, value):
     return comp
 
 
+def _map_filter_exp(entity_name, filter_exp, property_map):
+    if isinstance(filter_exp, (IntComparison, FloatComparison, StrComparison,
+                               ListComparison)):
+        # get the field
+        field = filter_exp.field
+        # add entity to field if it doesn't have one already
+        if ':' not in field:
+            field = f"{entity_name}:{field}"
```

Comment:
What will happen here if the user *isn't* using STIX objects and properties?  I still don't know OCSF that well, but if the statement is using OCSF it seems like we'll be creating an invalid field here.
Maybe if the new field name we created isn't in the map, we should use the original field (instead of the one with `{entity_name}:` prepended?

Reply:
> Maybe if the new field name we created isn't in the map, we should use the original field (instead of the one with {entity_name}: prepended?

I addressed this issue in commit https://github.com/opencybersecurityalliance/kestrel-lang/pull/434/commits/1a375dc2d5bb98b706c806624e1552c20a660b12

To fully address this comment, we need to provide a way, in  the Kestrel  configuration, to provide front-end mapping between multiple standards, in addition to the existing STIX to OCSF mapping.